src/main/java/org/prebid/server/floors/PriceFloorFetcher.java

```diff
@@ -65,14 +68,15 @@ public PriceFloorFetcher(ApplicationSettings applicationSettings,
                              Vertx vertx,
                              TimeoutFactory timeoutFactory,
                              HttpClient httpClient,
-                             JacksonMapper mapper) {
+                             JacksonMapper mapper, PriceFloorTestingProperties testingProperties) {
```

Comment:
Minor: according to project style, mapper should be last. And check all occurrences in this PR.